src/Symfony/Component/Console/Tests/Helper/OutputWrapperTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Helper;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Helper\OutputWrapper;
+
+class OutputWrapperTest extends TestCase
+{
+    public function testBasicWrap()
+    {
+        $wrapper = new OutputWrapper();
+        // Test UTF-8 chars + URL
+        $text = 'Árvíztűrőtükörfúrógép https://github.com/symfony/symfony Lorem ipsum <comment>dolor</comment> sit amet, consectetur adipiscing elit. Praesent vestibulum nulla quis urna maximus porttitor. Donec ullamcorper risus at <error>libero ornare</error> efficitur.';
+        $baseExpected = <<<EOS
```

Comment:
it doesn't look like you are using any variable inside so I would suggest the stricter version `'EOS'`